backend/app/adapter/sqldb/short_link_integration_test.go

```diff
@@ -47,6 +47,102 @@ type shortLinkTableRow struct {
 	twitterImageURL    *string
 }
 
+func TestShortLinkSql_UpdateOGMetaTags(t *testing.T) {
+	title1 := "title1"
+	description1 := "description1"
+	imageURL1 := "url1"
+	title2 := "title2"
+	description2 := "description2"
+	imageURL2 := "url2"
+
+	testCases := []struct {
+		name              string
+		tableRows         []shortLinkTableRow
+		alias             string
+		metaTags          metatag.OpenGraph
+		expectedShortLink entity.ShortLink
+	}{
+		{
+			name: "Update nil meta tags",
+			tableRows: []shortLinkTableRow{
+				{
+					alias:    "220uFicCJj",
+					longLink: "http://www.google.com",
+				},
+			},
+			alias: "220uFicCJj",
+			metaTags: metatag.OpenGraph{
+				Title:       &title1,
+				Description: &description1,
+				ImageURL:    &imageURL1,
+			},
+			expectedShortLink: entity.ShortLink{
+				Alias:    "220uFicCJj",
+				LongLink: "http://www.google.com",
+				OpenGraphTags: metatag.OpenGraph{
+					Title:       &title1,
+					Description: &description1,
+					ImageURL:    &imageURL1,
+				},
+			},
+		},
+		{
+			name: "Update non nil meta tags",
```

Comment:
```suggestion
			name: "Twitter tags provided",
```

Reply:
Please add test case for Open Graph Tags also

Reply:
Please add a test case where the update tags(metaTags) provided are the same as the ones that the short link has.

Reply:
The same tags test doesn't seem to provide much value here. It's optional for @kapillamba4  to decide to add this one.